README.md

```diff
@@ -73,9 +74,10 @@ Rules with a check mark (✅) are enabled by default while using the `plugin:cyp
 | :-- | :------------------------------------------------------------------------- | :-------------------------------------------------------------- |
 | ✅  | [no-assigning-return-values](./docs/rules/no-assigning-return-values.md)   | Prevent assigning return values of cy calls                     |
 | ✅  | [no-unnecessary-waiting](./docs/rules/no-unnecessary-waiting.md)           | Prevent waiting for arbitrary time periods                      |
-|     | [no-force](./docs/rules/no-force.md)                                       | Disallow using `force: true` with action commands                  |
+|     | [no-force](./docs/rules/no-force.md)                                       | Disallow using `force: true` with action commands               |
 |     | [assertion-before-screenshot](./docs/rules/assertion-before-screenshot.md) | Ensure screenshots are preceded by an assertion                 |
 |     | [require-data-selectors](./docs/rules/require-data-selectors.md)           | Only allow data-\* attribute selectors (require-data-selectors) |
+|     | [no-async-tests](./docs/rules/no-async-tests.md)                           | Prevent using async/await in Cypress test case                  |
```

Comment:
maybe set it on by default?

Reply:
Seeing as Cypress will error if you use an async test, I think that's a good idea.

Reply:
i could go either way. I wasn't sure what the policy was regarding default rules here